### DISCUSSION TITLE
Welcome to simulink-capi Discussions!

### DISCUSSION THREAD

**Discussion by danishbelal** [danishbelal/simulink-capi#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kilincceker** [comment]
Hello Danish,

This is Onur, a researcher at the University of Antwerp.

Thank you for this beneficial repo, which helped me a lot because I am also trying to test C++ code generated from Simulink model(s) using C API to make input and output variables testable.

I have some questions for you. Could you please get back to me at onur.kilincceker@uantwerpen.be?

My questions will be mainly on your repo and make use of it for my C++ code using C API.

I am looking forward to hearing from you.

Thank you in advance.

Kind regards,

Onur.
